### DISCUSSION TITLE
Is there a way to just render the Lexical post on React native?

### DISCUSSION THREAD

**Discussion by safa-kajabi** [facebook/lexical#4994] [Q&A]
We have web and mobile applications. We are using Lexical on web, we want to display the lexical post on mobile. How can I do it? Any suggestions

**Comment by akichim21** [comment]
https://github.com/facebook/lexical/discussions/2455#discussioncomment-2981557
Perhaps this is it.
But it would be better if lexical prepared a readonly package

**Comment by dohomi** [comment]
I found this package: https://www.npmjs.com/package/@atelier-disko/payload-lexical-react-renderer. You'd update the renderer to render your custom RN block/mark elements. Tried it with RN + Tamagui and works like a charm

**Reply by safa-kajabi** [reply]
Thank you! I will check that!
